### ISSUE TITLE
Error "'Tensor' object has no attribute 'sq'" during inference with my own data

### ISSUE THREAD

**Issue by Cyuanwen** [qcf-568/MIML#11]
Environment
PyTorch version: 1.13.0
Python version: 3.9

When running inference with modified data_dict (without GT masks) in casia1_infer.py, the script fails at line 187 in mmseg/models/decode_heads/test_rfb1ha1c.py with error:
```
'Tensor' object has no attribute 'sq'
```
The problematic code:
```
return lab_outs, F.softmax(cpred,dim=1)[:,1:2].sq  # Error occurs here
```

Could you please clarify:

1. Is .sq a custom method that requires specific initialization?

2. Should this be replaced with standard PyTorch operation like .square()?

3. Are there any version-specific dependencies for this functionality?

**Comment by qcf-568** [comment]
Hello! @Cyuanwen 

Sorry for the inconvenience. The ".sq" should actually be ".squeeze()". It appears that some characters were lost when the code was uploaded.

Thank you!
